docs/getting_started.rst

```diff
@@ -22,6 +21,7 @@ Quick Start
     calc = a3.Calculation(
         ensemble_size=5, # Use 5 (independently equilibrated) replicate runs
         slurm_config=a3.SlurmConfig(partition="<desired partition>"),  # Set your desired partition!
+        engine_config=a3.SomdConfig(),
```

Comment:
Meant please change the angled brackets if they are confusing - e.g. to CHANGEME.